blocks/shared-styles/index.js

```diff
@@ -1,24 +1,28 @@
-import Overline from './_children/overline';
 import Byline from './_children/byline';
+import Heading from './_children/headings/heading';
+import HeadingSection from './_children/headings/section';
+import Overline from './_children/overline';
+import PrimaryFont from './_children/primary-font';
 import PromoDate from './_children/promo-date';
 import PromoDescription from './_children/promo-description';
 import PromoHeadline from './_children/promo-headline';
-import PromoLabel from './_children/promo-label';
 import PromoImage from './_children/promo-image';
-import PrimaryFont from './_children/primary-font';
+import PromoLabel from './_children/promo-label';
 import SecondaryFont from './_children/secondary-font';
 import SmallPromoContainer from './_children/promo-helpers/small/container';
 import SmallPromoStyles from './_children/promo-helpers/small/styles';
 
 export {
-  Overline,
   Byline,
+  Heading,
```

Comment:
Robin will appreciate this alphabetization change

Reply:
Who doesn't like alphabetisation?

Reply:
![queer-eye-creeper](https://user-images.githubusercontent.com/13948125/118287240-424cdd00-b4a1-11eb-9e37-0a4381ab433c.gif)